content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: "Introducing Kueue"
+date: 
+slug:
+---
+
+**Authors:** Abdullah Gharaibeh (Google), Aldo Culquicondor (Google)
+
+For batch workloads, it does not make sense to focus only on pods. The partial execution of pods from multiple parallel batch jobs may 
```

Comment:
This feels a bit too sudden.

I think we could move this paragraph somewhere in the "why a new controller section" and add a paragraph here along these lines:

```markdown
Whether on prem or in the cloud, clusters have limited resources.
Organizations that run HPC, AI/ML and other batch
jobs want to maximize the utilization of these resources.
They would like to do so by over committing their kubernetes
clusters with jobs, hoping that kubernetes would start them in
an ordered and fair manner.
```